setup.py

```diff
@@ -95,5 +98,14 @@ def build_extension(self, ext):
         'Research': 'https://iic.jku.at/eda/research/quantum_simulation/',
     },
     python_requires='>=3.6',
-    setup_requires=['cmake>=3.14']
+    setup_requires=['cmake>=3.14'],
+    install_requires=[
+        'numpy>=1.19,<1.21',
+        'pandas>=1.1.4',
+        'networkx>=2.5',
```

Comment:
do we actually need networkx as a requirement for the project? I figured it is just used in the Graph State benchmarks.

Reply:
still to-do after everything else is finished